### PR TITLE
Thomas 20221014

### PR DESCRIPTION
**Pull request by tfrancart**
Few small improvements, most notably changes in SPARQL query generation to not use VALUES if only one value.

### CODE REVIEW

**Comment by tfrancart** [comment]
Can you review and merge this @SteinerPascal ? Thanks

File: static/default/scripts/initSparnatural.js [hunk 1]
```diff
@@ -61,4 +63,22 @@ sparnatural.addEventListener('componentLoaded',(e)=>{
...
+      JSON.parse(document.getElementById('query-json').value),
+      null,
+      2
+    );
+  $('#export-json').val(jsonString);
+  $('#exportModal').modal('show');       
+}
+
+
+document.getElementById('stored-queries').onchange = function() {
```
Comment (SteinerPascal) [inline static/default/scripts/initSparnatural.js:79]: What is this stored-quries? I don't see any element with this Id?

**Review by SteinerPascal — COMMENTED** [review]
In a new branch i have included federated queries in the generator. And refactored the whole generator. I have taken this query generation ordering to the new branch. I will exclud this change from the merge to not produce any merge conflicts

File: static/default/scripts/initSparnatural.js (hunk 1 above)
Comment (tfrancart) [reply static/default/scripts/initSparnatural.js]: This is a remainer of previous integration of sample queries dropdown. I have removed it.

**Review by SteinerPascal — APPROVED** [review]

**Pull request merged**